docroot/profiles/custom/cgov_site/modules/custom/cgov_promo/config/install/core.entity_view_mode.paragraph.ncids_group_flag_card.yml

```diff
@@ -0,0 +1,10 @@
+langcode: en
```

Comment:
@dev-rana-publicis  was going to respond to this, but he did not. This is being changed, but it is not `ncids_flag_card` because it is the display mode of the ncids_feature_card_XXX. We need the default view mode for them to be used for 2 column. We made the instance of the card in a group `ncids_group_card`, so we decided on making this be `ncids_flag_group_card` to match that naming convention.